node/src/blockchain/blockchain_interface.rs

```diff
@@ -149,7 +157,8 @@ impl BlockchainInterface for BlockchainInterfaceClandestine {
 
     fn retrieve_transactions(
         &self,
-        _start_block: u64,
+        _start_block: BlockNumber,
+        _latest_block: BlockNumber,
```

Comment:
Why is this called `_latest_block` here and `_end_block` above?

Reply:
Good question. I'll make it the same.